setup.py

```diff
@@ -0,0 +1,26 @@
+import re
+import setuptools
+import sys
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="lbt-butterfly",
+    use_scm_version = True,
+    setup_requires=['setuptools_scm'],
+    author="Ladybug Tools",
+    author_email="info@ladybug.tools",
+    description="A light python API for creating and running OpenFoam cases for CFD simulation.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/ladybug-tools/butterfly",
+    packages=setuptools.find_packages(exclude=["tests"]),
+    classifiers=[
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.6",
```

Comment:
It is not really Python 3 compatible. Let's remove this for now until we write some tests and make it Python 3 compatible!

Reply:
Done.